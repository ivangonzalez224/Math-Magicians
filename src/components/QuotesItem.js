import '../App.css';
import Quote from './Quote';

const QuotesItem = () => (
  <div className="main_quotes">
    <Quote />
  </div>
);

export default QuotesItem;
