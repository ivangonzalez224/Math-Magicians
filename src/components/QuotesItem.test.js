import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import QuotesItem from './QuotesItem';

describe('Quotes component', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <QuotesItem />
        </BrowserRouter>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});