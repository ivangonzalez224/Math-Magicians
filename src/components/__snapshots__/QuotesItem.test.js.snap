// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes component renders correctly 1`] = `
<div
  className="main_quotes"
>
  <p
    className="quote_content"
  >
    Loading...
  </p>
</div>
`;
